examples/chart/teleport-cluster/tests/__snapshot__/config_test.yaml.snap

```diff
@@ -179,6 +179,53 @@ matches snapshot for affinity.yaml:
     metadata:
       name: RELEASE-NAME
       namespace: NAMESPACE
+matches snapshot for aws-dynamodb-autoscaling.yaml:
+  1: |
+    apiVersion: v1
+    data:
+      teleport.yaml: |-
+        teleport:
+          log:
+            severity: INFO
+            output: stderr
+            format:
+              output: text
+              extra_fields: ["timestamp","level","component","caller"]
+          storage:
+            type: dynamodb
+            region: us-west-2
+            table_name: test-dynamodb-backend-table
+            audit_events_uri: ['dynamodb://test-dynamodb-auditlog-table']
+            audit_sessions_uri: s3://test-s3-session-storage-bucket
+            continuous_backups: false
+            auto_scaling: true
+            read_min_capacity: 5
+            read_max_capacity: 100
+            read_target_value: 50
```

Comment:
Interesting that this still renders as an integer despite being provided as a float. Running `teleport configure --test`gainst the config says it validates OK so maybe it doesn't have to be a float after all 🤷‍♂️